androidHyperskillApp/src/main/res/layout/item_project.xml

```diff
@@ -0,0 +1,139 @@
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textAppearance="@style/TextAppearance.AppTheme.Caption"
+            android:textColor="@color/color_on_surface_alpha_60"
+            android:text="@string/projects_list_project_graduate_title"
+            android:drawablePadding="6dp"
+            app:drawableStartCompat="@drawable/ic_graduate_project"
+            android:layout_marginStart="12dp"
+            app:layout_constraintStart_toEndOf="@+id/projectLevel"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/projectRating"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textAppearance="@style/TextAppearance.AppTheme.Caption"
+            android:textColor="@color/color_on_surface_alpha_60"
+            android:drawablePadding="6dp"
+            app:drawableStartCompat="@drawable/ic_star"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="4.7"/>
+
+        <TextView
+            android:id="@+id/projectTitle"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:textAppearance="@style/TextAppearance.AppTheme.Body1"
+            android:maxLines="2"
+            android:ellipsize="end"
+            android:layout_marginTop="12dp"
+            app:layout_constraintTop_toBottomOf="@+id/projectLevel"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            tools:text="Simple Chatty Bot (Python)Simple Chatty Bot (Python)Simple Chatty Bot (Python)Simple Chatty Bot (Python)"/>
+
+        <TextView
+            android:id="@+id/projectDuration"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:textAppearance="@style/TextAppearance.AppTheme.Caption"
+            android:textColor="@color/color_on_surface_alpha_60"
+            android:layout_marginTop="4dp"
+            android:layout_marginBottom="12dp"
+            app:layout_constraintTop_toBottomOf="@+id/projectTitle"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            tools:text="136 hours"
+            />
+
+        <androidx.constraintlayout.helper.widget.Flow
+            android:id="@+id/projectTags"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:constraint_referenced_ids="projectCompletedTextView,projectSelectedTextView,projectIdeRequiredTextView,projectBestRatingTextView,projectFastestToCompleteTextView"
+            app:flow_horizontalAlign="start"
+            app:flow_horizontalBias="0"
+            app:flow_verticalGap="4dp"
+            app:flow_wrapMode="chain"
```

Comment:
if badges not fitting in one line then this flow layout will wrap the current line and move the next badge item to the new line?

Reply:
<img width="323" alt="Screenshot 2023-05-17 at 17 55 54" src="https://github.com/hyperskill/mobile-app/assets/25664794/661a02e0-a0da-4a74-8b09-773cdc0f8514">

Reply:
Really nice 👍